packages/flutter/lib/src/material/input_border.dart

```diff
@@ -116,6 +116,99 @@ class _NoInputBorder extends InputBorder {
   }
 }
 
+/// Creates an [InputBorder] that has no painted outline and rounded corners.
```

Comment:
`Creates an [InputBorder] that has rounded corners but no painted outline.`

Reply:
Maybe mention that it's intended to be used with a filled background. And link to one of the images in Crane.